hack/istio/download-istio.sh

```diff
@@ -17,7 +17,7 @@
 ##############################################################################
 
 # The version used by the getLatestIstio script - if empty, gets the latest version
-ISTIO_VERSION=
+ISTIO_VERSION=1.0.0-snapshot.1
```

Comment:
put a big loud TODO here to remind us that when 1.0 is released, go back to empty string. somethign like:

# TODO when Istio 1.0 is released, remove the version string to the right of the equal side - we want this set to empty string so by default we pick up the latest version.

Reply:
done